warning: format string is not a string literal

CRAN has asked for a fix the following new problem

(from https://cran.r-project.org/web/checks/check_results_httpuv.html)


```
Version: 1.6.12
Check: whether package can be installed
Result: WARN 
  Found the following significant warnings:
    webapplication.cpp:223:16: warning: format string is not a string literal (potentially insecure) [-Wformat-security]
  See ‘/home/hornik/tmp/R.check/r-devel-clang/Work/PKGS/httpuv.Rcheck/00install.out’ for details.
  * used C compiler: ‘Debian clang version 17.0.5 (1)’
  * used C++ compiler: ‘Debian clang version 17.0.5 (1)’
Flavor: [r-devel-linux-x86_64-debian-clang](https://www.r-project.org/nosvn/R.check/r-devel-linux-x86_64-debian-clang/httpuv-00check.html)
```

--- a/NEWS.md
+++ b/NEWS.md
@@ -1,5 +1,7 @@
 # httpuv (development version)
 
+* Closed #388: Fix R CMD check warning re error() format strings (for r-devel). (#389)
+
 # httpuv 1.6.12
 
 * New `runStaticServer()` provides a convenient interface for serving a directory of static files. (#380)

--- a/src/webapplication.cpp
+++ b/src/webapplication.cpp
@@ -220,7 +220,7 @@ std::shared_ptr<HttpResponse> listToResponse(
       Rcpp::as<bool>(response["bodyFileOwned"])
     );
     if (ret != FDS_OK) {
-      REprintf(pFDS->lastErrorMessage().c_str());
+      REprintf("%s", pFDS->lastErrorMessage().c_str());
       return error_response(pRequest, 500);
     }
     pDataSource = pFDS;

